packages/components/src/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ThemeProps } from '@t3n/theme';
+import {
+  SocialXing,
+  SocialLinkedin,
+  SocialTwitter,
+  SocialGithub,
+  SocialEmail,
+} from '@t3n/icons';
+import { Card } from '../Card';
+import { Heading } from '../Heading';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { Avatar } from '../Avatar';
+import { Grid } from '../Grid';
+import { GridItem } from '../GridItem';
+import { Badge } from '../Badge';
+import { Button } from '../Button';
+
+const StyledCard = styled(Card)`
+  position: relative;
+  height: 100%;
+`;
+
+const StyledBadge = styled(Badge)`
+  position: absolute;
+  top: 8px;
+  right: 8px;
+`;
+
+const SocialButton = styled(Button)`
+  border-radius: 50%;
+  border: 0;
+  background-color: ${({ theme }: ThemeProps) =>
+    theme.colors.background.secondary};
+  width: 24px;
+  height: 24px;
+  align-items: center;
+  margin: 0 0.5rem 0 0;
+  padding: 5px;
+
+  &:hover:not(:disabled) {
+    transform: scale(1);
+    background-color: ${({ theme }: ThemeProps) =>
+      theme.colors.background.inverse};
+  }
+
+  &:hover > svg {
+    fill: #fff;
+  }
+`;
+
+export type SocialLinkType =
+  | 'TWITTER'
+  | 'GITHUB'
+  | 'XING'
+  | 'HOMEPAGE'
+  | 'LINKEDIN';
+
+type SocialLinks = { url: string; type: SocialLinkType }[];
```

Comment:
Können wir den Type im Singular machen? Dann ist er genierischer. Das er als Array genutzt wird sollte der jeweils "nutzende" Type machen. Also in diesem Fall UserCardProps und SocialLinksProps :)